stress/src/metrics.rs

```diff
@@ -20,5 +28,19 @@ fn main() {
 }
 
 fn test_counter() {
-    COUNTER.add(1, &[]);
+    let mut rng = rand::thread_rng();
+    let len = ATTRIBUTE_VALUES.len();
+    let index_first_attribute = rng.gen_range(0..len);
+    let index_second_attribute = rng.gen_range(0..len);
+    let index_third_attribute = rng.gen_range(0..len);
```

Comment:
Initializing/generating random numbers could have significant overhead, need to see if there is easy way not to include this overhead in throughput calculation. If this is not easy to fix in this PR, we can add a TODO to take care in future.

Reply:
I think its inevitable that the cost must be paid to generate random number! An alternate could be to just call attribute[1,2,3,...6,1,2,3...] sequentially.
I'll check if this can be optimized further.

Reply:
You're using the `ThreadRng` here, though, which is much slower than necessary for this purpose because it is documented to be secure. Try using a `SmallRng` instead.

Reply:
Perhaps we could also generate a sequence of three value tuples before hand (outside of this function) and then call test_counter with each tuple sequentially?

Reply:
Moving to `SmallRng` improved throughtput in my machine from ~2-3 M/sec to ~6 M/sec.

Reply:
> Perhaps we could also generate a sequence of three value tuples before hand (outside of this function) and then call test_counter with each tuple sequentially?

I was thinking the same thing, because at some level we're testing the random generator in this stress test but if all the attributes are before it better tests the machinery.